### PR TITLE
APIS-2664 - add time parse operation

### PR DESCRIPTION
**Pull request by abaehre**

### CODE REVIEW

**Comment by paas-api-pr-manager** [comment]
Jira Tickets included in PR are:

* [APIS-2664](https://jira.gannett.com/browse/APIS-2664): Leverage XMLTransform to update livescores in sports-aggregator
```
XMLTransform has been added to the jstransform library. Leverage it to convert the data coming in from SDI to the proper schema for the livescores data type.
This will require updating docs, learning XPath and delving into some jstransform code.

All the different transforms should still be available, but if more need to be added a new ticket will need to be created.

AC - 
XMLTransform is leveraged to change the data for livescores into the schema found in content-api-sport. That data is then passed to content-ingestor-api.
```

File: transform/operations.go [hunk 1]
```diff
@@ -219,6 +220,32 @@ func (s *split) transform(raw interface{}) (interface{}, error) {
...
+	}
+
+	t.Args = args
+	return nil
+}
+
+func (t *timeParse) transform(raw interface{}) (interface{}, error) {
+	in, ok := raw.(string)
+	if !ok {
+		return nil, errors.New("split only supports strings")
```
[the remark this answers is not in the document]
Comment (abaehre) [reply transform/operations.go]: damn thought I got all these...

File: transform/operations_test.go [hunk 2]
```diff
@@ -337,3 +338,39 @@ func TestSplit(t *testing.T) {
...
+		},
+		{
+			description: "Missing arg",
+			args:        map[string]string{"format": time.RFC3339},
+			in:          "2019-05-16T21:00:00-04:00",
+			wantInitErr: true,
+		},
+		{
+			description: "Too many args",
+			args:        map[string]string{"format": time.RFC3339, "layout": "2006-01-02", "cookies": "failure"},
```
[the remark this answers is not in the document]
Comment (abaehre) [reply transform/operations_test.go]: /shrug I guess not

**Comment by abaehre** [comment]
@tkuhlman @ahornerr bump - whenever you get a chance.

**Pull request merged**